src/native/minipal/CMakeLists.txt

```diff
@@ -36,6 +37,10 @@ endif()
 add_library(minipal STATIC)
 target_link_libraries(minipal PRIVATE minipal_objects)
 
+if(CLR_CMAKE_TARGET_ANDROID)
+  target_link_libraries(minipal PRIVATE log)
+endif(CLR_CMAKE_TARGET_ANDROID)
```

Comment:
```suggestion
if(CLR_CMAKE_HOST_ANDROID)
  target_link_libraries(minipal PRIVATE log)
endif(CLR_CMAKE_HOST_ANDROID)
```

Reply:
I thought the `_HOST_` values were for the machine doing the compiling. I would have assumed `_TARGET_` is correct. Can you clarify the distinction?

Reply:
HOST is the machine the code we are building will run on. TARGET is the machine the running code targets. Usually the same, but TARGET is different for the cross-targeting JITs and cross-OS DAC.

Reply:
Right, so my understanding is correct. Now onto the second part. This is for the `minipal`, neither cross-targeting JIT nor cross-OS DAC - at least specifically. So why is `_HOST_` appropriate here? We're not building the `minipal` on an Android host, so why is it special?

Reply:
> I thought the _HOST_ values were for the machine doing the compiling.

HOST is not the BUILD configuration. It is NOT the machine you are building on.

When you are building for Android on macOS then you will build two versions of minipal. One of them will be for the cross-compiler tools and cross-DAC (HOST == BUILD == osx-arm64, TARGET == android-arm64). The other one will be for the device (HOST == TARGET == android-arm64, BUILD == osx-arm64).

Reply:
Let's take our build machines building the .NET SDK for linux-arm, in particular the JIT for linux-arm64.

We have three configurations:

Build: linux-x64 (Azure Linux 3.0)
Host: linux-arm (the cross rootfs with the headers/libraries we build against)
Target: linux-arm64

Reply:
FWIW, this is de-facto standard, not something that we have invented. For example, check https://gcc.gnu.org/onlinedocs/gccint/Configure-Terms.html

Reply:
changed to use HOST instead of TARGET in cmake file.